Sign Agreement

mfaerevaag opened the pull request.

sangkilc commented:
Please do not change Agreement.md. Revert your change, and commit again with a new file.

sangkilc commented:
Please resolve this problem!

mfaerevaag commented:
I am so sorry! I don't understand how I haven't understood that I was the one that was missing to resolve his issue... Thanks @soomin-kim for telling me.

I rebased, resolved the issue and cleaned up the history so it can be safely merged.

Again, sorry for this..
